Source/WTF/wtf/haiku/RunLoopHaiku.cpp

```diff
@@ -100,7 +100,7 @@ void RunLoop::run()
 	looper->LockLooper();
 	looper->AddHandler(current().m_handler);
 	looper->UnlockLooper();
-
+	
```

Comment:
i will get the spacing and weird debug messages out as now my only aim is to get it rendering 😄